### PR TITLE
💩generated file actions #72661

### LINKED ISSUE #72661: Tests should be disabled in forks when they are expected to fail
### 🐛 Describe the bug

Currently many tests run in forks but they are expected to fail due to lack of credentials in forks.

This is a bit annoying to contributors:

A failed job example: https://github.com/ppwwyyxx/pytorch/runs/5137367784?check_suite_focus=true

I think github action should be configured to not run these in forks.

### Versions

latest as of 02/09

cc @seemethere @malfet @pytorch/pytorch-dev-infra

### PR DESCRIPTION
**Pull request by arunavabasucom**
helps progress the issue 72661

I fix the man file. Please review @janeyx99  and give me feedback.

### CODE REVIEW

**Comment by arunavabasu-03** [comment]
hey @janeyx99  please review that.

File: .github/templates/ios_ci_workflow.yml.j2 [hunk 1]
```diff
@@ -43,7 +43,10 @@ jobs:
   build:
     # NOTE: These builds will not run successfully without running on `pytorch/pytorch` due to the limitations
     #       of accessing secrets from forked pull requests and IOS' dependency on secrets for their build/test
-    if: ${{ github.event_name == 'push' || github.event.pull_request.head.repo.full_name == github.repository }}
+    if: ${{ github.event_name == 'push' || (github.event.pull_request.head.repo.full_name == github.repository && github.repository_owner == 'pytorch') }}
+    runs-on: macos-10.15
+    timeout-minutes: 240
+    env:
```
Comment (seemethere) [inline .github/templates/ios_ci_workflow.yml.j2:49]:
```suggestion
```

I don't think these changers are actually necessary and seem to be a duplication of the lines below these lines

**Review by seemethere — COMMENTED** [review]
Also looks like you need to regenerate the workflows using `.github/regenerate.sh`

**Comment by seemethere** [comment]
Could you add a PR title that's representative of what's in this PR?

**Comment by janeyx99** [comment]
Also, since this only addresses one of the workflows in the linked issue, I'd recommend removing the "Fixes #" in the PR body and writing something like "helps progress the issue" instead. That way, the issue won't be wrongfully closed when this PR lands.

**Pull request closed without merging**